kvdb/etcd/readwrite_bucket.go

```diff
@@ -66,6 +70,29 @@ func (b *readWriteBucket) ForEach(cb func(k, v []byte) error) error {
 	return nil
 }
 
+// ForAll is an optimized version of ForEach with the limitation that no
+// additional queries can be executed within the callback.
```

Comment:
nit: update with the parent PR's comment that uses the slightly more explicit `NOTE: ` notation?

Reply:
Sorry my original source of the comment was from the implementation, added the note from the interface here too.